test/unit/Signer.test.js

```diff
@@ -66,10 +94,11 @@ describe('Signer', () => {
                 signedRequest.publisherAddress,
                 signedRequest.signature,
             )
-            Signer.verifyStreamMessage(streamMessage)
+            Signer.verifyStreamMessage(streamMessage, new Set([signedRequest.publisherAddress]))
         })
 
         it('Should throw if incorrect signature', async () => {
```

Comment:
Doesn't need to be `async`?

Reply:
I dont think so because I don't see any `async` function inside `verifyStreamMessage`. `web3.eth.accounts.recover` is not `async`.